app/src/androidTest/java/com/google/android/perftesting/PerfTestTemplate.java

```diff
@@ -50,13 +50,13 @@
 
     public EnableTestTracing mEnableTestTracing = new EnableTestTracing();
 
-    public EnablePostTestDumpsys mEnablePostTestDumpsys = new EnablePostTestDumpsys();
+    public EnablePostTestDumpsys mEnablePostTestDumpsys = new EnablePostTestDumpsys(10);
```

Comment:
The usage of `EnablePostTestDumpsys` test rule seems weird. How about creating a separate test rule, such as`MeasureJankPercentage`, dedicated to measurement of jank frames?